src/options.rs

```diff
@@ -415,13 +415,17 @@ pub struct UI {
 #[derive(clap::Args, Debug, Clone)]
 pub struct Info {
    #[arg(long, value_parser=[
+        "install-dir",
+        "binary-dir",
```

Comment:
I went with `binary-dir` over the discussed `bin-dir`. The longer form just seemed to fit with the other options here better than the shorter form.

Reply:
Furthermore, what we're actually interested in for the npx wrapper is the _install_ directory (which is confusingly called `binary_path`)

Reply:
Hm. Wait. `install-dir` is what we actually want here. Why do we need the directory of the current binary?

Reply:
Mostly just because I started with `binary-dir` before I realized what we want is `install-dir`, so I just kept it. Also, this makes it so that everything you get in the `info` command* is available one-by-one using `--get`.

*Well, we actually show the path to the binary rather than the directory it's in in the info table, but this still seems right.

---

I can just remove the `binary-dir` option here if you'd prefer to only expose what we need.

Reply:
This is what the info table now shows for me running this with cargo on macOS, which shows the difference between `binary` and `install` more obviously:

```
EdgeDB uses the following local paths:
┌───────────────┬────────────────────────────────────────────────────────────┐
│ Cache         │ /Users/scotttrinh/Library/Caches/edgedb/                   │
│ Config        │ /Users/scotttrinh/Library/Application Support/edgedb/      │
│ Custom Binary │ /Users/scotttrinh/edgedb/edgedb-cli/target/debug/edgedb    │
│ Install       │ /Users/scotttrinh/Library/Application Support/edgedb/bin/  │
│ Data          │ /Users/scotttrinh/Library/Application Support/edgedb/data/ │
│ Service       │ /Users/scotttrinh/Library/LaunchAgents/                    │
└───────────────┴────────────────────────────────────────────────────────────┘
```

Reply:
I think `binary-dir` is redundant.  You can always use `which edgedb` if you really need to find where the binary is located.